### ISSUE TITLE
[Windows] [Packager] Syntax error in setup_env.bat breaks packager on Windows in 0.14.0-rc

### ISSUE THREAD

**Issue by rigdern** [facebook/react-native#3757]
### Repro Steps

The packager in 0.14.0-rc doesn't work on Windows.

When running the packager:

```
> node node_modules\react-native\local-cli\cli.js start
```

I get the following error:

```
'/**' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
child_process.js:464
    throw err;
    ^

Error: Command failed: C:\Users\adcom\Source\Repos\client_web_skypex-prototypes\ReactNativeCX\node_modules\react-native\local-cli\setup_env.bat
'/**' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.
'*' is not recognized as an internal or external command,
operable program or batch file.

    at checkExecSyncError (child_process.js:441:13)
    at Object.execFileSync (child_process.js:461:13)
    at run (C:\Users\adcom\Source\Repos\client_web_skypex-prototypes\ReactNativeCX\node_modules\react-native\local-cli\cli.js:59:16)
    at Object.<anonymous> (C:\Users\adcom\Source\Repos\client_web_skypex-prototypes\ReactNativeCX\node_modules\react-native\local-cli\cli.js:145:3)
    at Module._compile (module.js:434:26)
    at Object.Module._extensions..js (module.js:452:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Function.Module.runMain (module.js:475:10)
    at startup (node.js:117:18)
```

### Cause

It looks like [setup_env.bat](https://github.com/facebook/react-native/blob/ca7a8b271a92d6be3c7b56f35f6091c1c6a66a3c/local-cli/setup_env.bat) uses the wrong syntax for commenting which is causing the problem. The correct syntax for comments in `.bat` files is:

```
:: This line is a comment
```

whereas the file is currently using `/* ... */`.

**Comment by martinbigio** [comment]
Sorry about that, just fixed it on 89f1e67. @mkonicek could we cherry pick that rev on the rc? :)

**Comment by arhcstolen** [comment]
This problem is still present on 0.14-stable

**Comment by rigdern** [comment]
As @arhcstolen mentioned, it doesn't look like this change made it into the 0.14.0 release. Others have been hitting this bug:
  - #3934
  - #3956

**Comment by ide** [comment]
I went ahead and published RN 0.14.1 to npm, which contains 89f1e6732ec0f18ae6d09cee3c8795f5a752514f.

**Comment by rigdern** [comment]
@ide Awesome, thanks! I can confirm it works now.
